src/session.php

```diff
@@ -37,6 +38,10 @@
             }
         }
     }
+    if (\PHP_VERSION_ID >= 70100) {
+        // Forward compatibility.
+        unset($session_config['entropy_length']);
+    }
```

Comment:
This configuration directive is removed in PHP 7.1.0, so we might as well not pass it.